src/app/modules/spcp/spcp.routes.ts

```diff
@@ -95,3 +95,14 @@ CorppassLoginRouter.get(
   loginParamsMiddleware,
   SpcpController.handleLogin(FormAuthType.CP),
 )
+
+// Handles SingPass JWKS requests
+export const SpOidcJwksRouter = Router()
```

Comment:
should this be its own file in our root `/routes` directory? to better follow the mapping of routes in our application.

Reply:
yes, good idea!